src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Books from './components/books';
+import Header from './components/header';
+import Category from './components/category';
+
+function App() {
+  return (
+    <div className="container">
+      <Router>
+        <Header />
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/category" element={<Category />} />
+        </Routes>
```

Comment:
✅ Has 2 `<Route>`s, one for the BookList component and the other for the Categories component 👍🏻.